python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -166,6 +166,41 @@ def test_library_management_path_prefix(moto_s3_uri_incl_bucket, boto_client):
         _lib = ac["pytest_test_lib"]
 
 
+@pytest.mark.parametrize("arctic_library", [["use_tombstones", "sync_passive.enabled"]], indirect=True)
+def test_read_meta_batch_with_tombstones(arctic_library):
```

Comment:
The "implementation" has no logic, so probably superfluous to test.

Reply:
Removed that and improved the test